include/seqan3/alignment/matrix/detail/aligned_sequence_builder.hpp

```diff
@@ -137,6 +95,21 @@ class aligned_sequence_builder
     {}
     //!\}
 
+    /*!\brief Sets the column offset corresponding to the upper diagonal of the band within the matrix.
+     * \param[in] offset The offset to set.
+     *
+     * \details
+     *
+     * The offset corresponds to the cell where the upper diagonal intersects with the first row of the matrix after
+     * trimming the sequences. This option is only necessary if the matrix was computed with a banded alignment.
```

Comment:
No, this must be handled here. We don't know the end of the traceback before hand. Unless we want to parse the trace two times. But we don't want to do that. For structural and performance reasons. So since the trace can be only covering a slice of the original sequences the actual alignment is sliced (trimmed might be really the wrong term here).

Reply:
The trimming part yes, but my initial comment was directed at the offset only used in the banded case. And that - you said - was removed, so that should have made this function obsolete.